rust/share/agama.pam

```diff
@@ -0,0 +1,8 @@
+#%PAM-1.0
```

Comment:
Again, I have no clue what these lines mean. Maybe a short description or a link to some documentation here would be nice...